submissions/DmytryjK/HTML-CSS-Popup/css/components/fonts.css

```diff
@@ -0,0 +1,14 @@
+@font-face {
+  font-family: "inter-regular";
+  src: url("../fonts/Inter2/converted/Inter-Regular.eot");
+  src: url("../fonts/Inter2/converted/Inter-Regular.woff") format("woff"), url("../fonts/Inter2/converted/Inter-Regular.otf") format("opentype"), url("../fonts/Inter2/converted/Inter-Regular.ttff") format("truetype");
+  font-weight: 400;
+  font-style: normal;
+}
+@font-face {
+  font-family: "inter-bold";
+  src: url("../fonts/Inter2/converted/Inter-Bold.eot");
+  src: url("../fonts/Inter2/converted/Inter-Bold.woff") format("woff"), url("../fonts/Inter2/converted/Inter-Bold.otf") format("opentype"), url("../fonts/Inter2/converted/Inter-Bold.ttff") format("truetype");
+  font-weight: 700;
+  font-style: bold;
+}
```

Comment:
Add an empty line at the end of the file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

Reply:
fixed